### PR TITLE
simplefs: add a method and cmd for setting the fs debug level

### PR DESCRIPTION
**Pull request by strib**
cc @patrickxb on the vdebug changes.

To improve KBFS performance, we want to turn off most logging by default, but still let the user set it at runtime.  To do that, we'll move a bunch of logging to vdebug, and then this new command/RPC will let users set it at runtime.

That required making the vdebug logger goroutine-safe, and while I was in there I cleaned up the string constants a bit.

Issue: KBFS-4053

### CODE REVIEW

File: go/kbfs/libkbfs/config_local.go [hunk 1]
```diff
@@ -916,10 +917,11 @@ func (c *ConfigLocal) MakeLogger(module string) logger.Logger {
 
 // MakeVLogger implements the logMaker interface for ConfigLocal.
 func (c *ConfigLocal) MakeVLogger(module string) *libkb.VDebugLog {
-	// No need to lock since c.loggerFn is initialized once at
-	// construction. Also resetCachesWithoutShutdown would deadlock.
```
Comment (patrickxb) [inline go/kbfs/libkbfs/config_local.go]: This `resetCachesWithoutShutdown` deadlock is no longer an issue?

Comment (strib) [reply go/kbfs/libkbfs/config_local.go]: That comment had been copy/pasted from the function above this one, and doesn't apply to it.

File: go/client/cmd_simplefs_set_debug_level.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
```
Comment (songgao) [inline go/client/cmd_simplefs_set_debug_level.go]: 2019

File: go/client/cmd_simplefs_set_debug_level.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	libkb.Contextified
+	level string
+}
+
+// NewCmdSimpleFSSetDebugLevel creates a new cli.Command.
+func NewCmdSimpleFSSetDebugLevel(
+	cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "set-debug-level",
+		ArgumentHelp: "<integer_level>",
```
Comment (songgao) [inline go/client/cmd_simplefs_set_debug_level.go]: I wonder if it'd make sense to just have this accept strings `mobile`, 'vlog1', etc.

File: go/libkb/vdebug.go [hunk 4]
```diff
@@ -32,26 +35,57 @@ const (
 	VLog1
 	VLog2
 	VLog3
+
+	VLogNoneString       = "mobile"
```
Comment (songgao) [inline go/libkb/vdebug.go]: Is the plan to have "mobile" equivalent to None forever?

File: go/client/cmd_simplefs_set_debug_level.go (hunk 3 above)
Comment (strib) [reply go/client/cmd_simplefs_set_debug_level.go]: I didn't want to expose those to users when we ask them to run this command.  Numbers seemed better to me.

File: go/libkb/vdebug.go (hunk 4 above)
Comment (strib) [reply go/libkb/vdebug.go]: No clue -- I'm just rearranging what was already there.  I don't think it matters right now.

**Pull request merged**